test/bigtest/tests/pci-title-info-test.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { describe, beforeEach, it } from '@bigtest/mocha';
+import chai from 'chai';
+import spies from 'chai-spies';
+import { StaticRouter as Router } from 'react-router-dom';
+import { mountWithContext } from '../helpers/mountWithContext';
+
+import TitleInfo from '../../../src/components/EResourceSections/TitleInfo';
+
+import { monographResource, serialResource } from './resources';
+import PCITitleInfoInteractor from '../interactors/pci-title-info';
+
+chai.use(spies);
+const { expect } = chai;
+
+describe('PCI title information', () => {
+  const interactor = new PCITitleInfoInteractor();
+
+  describe('monograph resource', () => {
+    beforeEach(async () => {
+      await mountWithContext(
+        <Router context={{}}>
+          <TitleInfo
+            data={{ eresource: monographResource }}
+          />
+        </Router>
+      );
+    });
+
+    it('should render expected title', () => {
+      expect(interactor.titleInstanceName).to.equal(monographResource.pti?.titleInstance?.name);
```

Comment:
_Never_ use an optional chain in an expected value. If the chain snaps and gives you `undefined`, you almost never want whatever you're `expect()`ing to be `undefined` as well. (In the rare case you do, you should use `.to.be.undefined()` or something.)

Sometimes with tests like this it almost makes more sense to just write out what you really want so it's very clear what should happen: `expect(interactor.titleInstaanceName).to.equal("Advances in Arithmetic")`